src/concurrency/timestamp_ordering_transaction_manager.cpp

```diff
@@ -397,83 +163,188 @@ bool TimestampOrderingTransactionManager::AcquireOwnership(
 // It should not be called if the tuple is in the write set as commit and abort
 // will release the write lock anyway.
 void TimestampOrderingTransactionManager::YieldOwnership(
-    UNUSED_ATTRIBUTE Transaction *const current_txn, const oid_t &tile_group_id,
+    UNUSED_ATTRIBUTE Transaction *const current_txn, 
```

Comment:
it's used in assertion. the compiler will complain when compiling under release mode.